xhype/xhype/src/bios.rs

```diff
@@ -0,0 +1,356 @@
+use crate::mach::MachVMBlock;
```

Comment:
You need to put license header in each file you create. 

/* SPDX-License-Identifier: GPL-2.0-only */
will do.

Reply:
license headers are added.